src/test/java/seedu/address/testutil/MarkTestUtil.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.testutil;
```

Comment:
its to prevent confusion with the TypicalStudents used by other commands. I needed a specific set of students to test the mark commands, and I didn't want to break everything by changing the common TypicalStudents